gateway/gateway.go

```diff
@@ -172,7 +173,6 @@ type HandleT struct {
 	trackSuccessCount int
 	trackFailureCount int
 
-	webRequestBatchCount                                       uint64
```

Comment:
@chandumlg we removed this global counter which was used for populating the `batch_id` parameter. Something to do with pending events?